Wikipedia/UI-V5/WMFNearbySectionController.m

```diff
@@ -79,8 +82,8 @@ - (NSString*)footerText {
 }
 
 - (NSArray*)items {
-    if ([self.nearbyResults.results count] > 0) {
-        return self.nearbyResults.results;
+    if ([self.viewModel.locationSearchResults.results count] > 0) {
```

Comment:
The friends of my friends… are not my friends. 

To much touching here… really need to provide an API on the view model to get what you need.